### PR TITLE
Add wu and rokita to whitelist

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

https://github.com/AlexsLemonade/ScPCA-admin/issues/834

## Purpose/Implementation Notes

Adds `wu` and `rokita` to whitelist for loading data.

## Types of changes

- New feature (non-breaking change which adds functionality)

## Functional tests

n/a

## Checklist

- [X] Lint and unit tests pass locally with my changes

## Screenshots

n/a

### CODE REVIEW

File: api/scpca_portal/management/commands/load_data.py [hunk 1]
```diff
@@ -24,6 +24,8 @@
     "murphy_chen",
     "pugh",
     "teachey_tan",
+    "wu",
+    "rokita",
```
Comment (jaclyn-taroni) [inline api/scpca_portal/management/commands/load_data.py:28]:
```suggestion
    "rokita"
```

**Review by jaclyn-taroni — APPROVED** [review]
Trailing comma, but 👍🏻  otherwise

File: api/scpca_portal/management/commands/load_data.py (hunk 1 above)
Comment (jaclyn-taroni) [reply api/scpca_portal/management/commands/load_data.py:28]: This might be convention based on the diff, so feel free to ignore me!

Comment (davidsmejia) [reply api/scpca_portal/management/commands/load_data.py:28]: Yeah it's convention so we don't have to edit the last item's line when adding/sorting a list. Our linter does require it.

**Pull request merged**